fence/agents/lib/fencing.py.py

```diff
@@ -763,6 +763,9 @@ def set_multi_power_fn(connection, options, set_power_fn, get_power_fn, retry_at
 
 		for _ in range(int(options["--power-timeout"])):
 			if get_multi_power_fn(connection, options, get_power_fn) != options["--action"]:
+				if "no_status" in options["device_opt"]:
```

Comment:
If there is no 'status' there should be no 'return False', the status action should not be called at all.

Reply:
Well, true in theory. But as implemented till now it is called and as the interface for setting a value doesn't foresee a return value this is an admittedly ugly way of still being able to convey a return value somehow for these cases where there is no real status-query available. I don't see an easy way to achieve that differently without breaking the interface.

Reply:
I agree that it might require more work in fencing library but @oalbrigt should be able to fix that fast. We already have fence agent that have no_status (rcd_serial) and it looks like it works. But it was tested just by users.

Reply:
I had of course checked rcd_serial! Reason why that is working is that it doesn't support setting an explicit value but just rebooting. Haven't found a fence-agent that supports explicit setting + no_status. With this kind of fake-status-call we can support that flavor without breaking the interface. 
Of course we could support a setter-function with return-value as enhancement of fence_action. (like result = fence_action(..., set_power_with_status=...)
Is that what you had in mind. I can try and see how clumsy that would finally look.

Reply:
That's a good point. We expect that verification cannot work with method=cycle but it was intended for reboot only originally.